docs/how-to-guides/use-init-flags.md

```diff
@@ -0,0 +1,186 @@
+---
+id: use-init-flags
+title: How to use init flags with Docker
+sidebar_label: Use init flags with Docker
+---
+
+With `init-file` and `init-data-file` configuration flags, you can execute queries from the CYPHERL file which need to be executed before or immediately after the Bolt server starts, respectively. The CYPHERL file, that the `init-file` flag is pointing to, is usually used to create users and set their passwords, so that no one except for the authorized users can't access data in the first run. The CYPHERL file, that the `init-data-file`is pointing to, is usually used to populate the database. If you plan to run Memgraph in Docker, then you have to make sure that the files `init-file` and `init-data-file` configuration flags are referring to are inside the container before Memgraph starts. In Docker, files can't be directly copied into a container before it's started because the filesystem of the container doesn't exist until it's actually running. However, you can tackle this by using a Dockerfile.
```

Comment:
```suggestion
With `init-file` and `init-data-file` configuration flags, you can execute queries from a CYPHERL file that need to be executed before or immediately after the Bolt server starts. The CYPHERL file the `init-file` flag points to is usually used to create users and set their passwords allowing only authorized users to access the data in the first run. The CYPHERL file the `init-data-file` points to is usually used to populate the database. 

If you will run Memgraph with Docker, make sure that the `init-file` and `init-data-file` configuration flags are referring to the files inside the container before Memgraph starts. Files can't be directly copied into a container before it's started because the filesystem of the container doesn't exist until it's actually running. However, you can tackle this by using a Dockerfile.
```

Reply:
or mounting a volume?

Reply:
link to config page

Reply:
Not possible with volume mount, but there is one other way, related to https://github.com/memgraph/docs/pull/924

Reply:
I will not be adding another way of running with init flags, because I experienced some issues and after syncing with Ante, we think it's fine to leave it as it is and update sometimes later if needed.